app/models/competency.rb

```diff
@@ -0,0 +1,14 @@
+class Competency < ActiveRecord::Base
+  # Relationships
+  # has_many :indicators
+
+  # Validations
+  validates_presence_of :name
+
```

Comment:
should you also validate the presence of description? Kim made it seem like all competencies will come with short descriptions.